tests/testthat/test-guides.R

```diff
@@ -524,3 +524,18 @@ test_that("coloursteps guide can be styled correctly", {
     p + guides(colour = guide_coloursteps(ticks = TRUE))
   )
 })
+
+test_that("a warning is generated when guides(<scale> = FALSE) is specified", {
+  df <- data_frame(x = c(1, 2, 4),
+                   y = c(6, 5, 7))
+
+  expect_warning(g <- guides(colour = FALSE), "`guides(<scale> = FALSE)` is deprecated.", fixed = TRUE)
+  expect_equal(g[["colour"]], "none")
+
+  # manually create a invalid guide
+  g[["colour"]] <- FALSE
```

Comment:
I would change the test so it uses an existing scale function. The current version seems a bit contrived.

Reply:
Sure.